docs/api.md

```diff
@@ -83,6 +84,22 @@ attributes key like `<tag attributes={{'data-custom-attr': 'foo'}} />`.
 
 - - -
 
+### <a id="JSX"></a> `JSX`
+
+A JSX plugin provide the transformation of the open standard JSX with the use of Babel.
+Firstly install this [babel-plugin-transform-jsx](https://www.npmjs.com/package/babel-plugin-transform-jsx) plugin,
+in your babel configuration use the following:
+```
+{
+ "plugins": [["babel-plugin-transform-jsx", { "module": "@cycle/dom/jsx" }]]
+}
+```
+Note there is now no need to add `\/\* @jsx *\/\` at the top of the files using JSX.
+
+For more information read the documentations [here]((https://www.npmjs.com/package/babel-plugin-transform-jsx)).
+
+- - -
+
```

Comment:
@Cmdv did you write to this markdown document directly or did you generate it using `npm run docs`? We can't do the former.